lib/Dialect/FIRRTL/FIRRTLTypes.cpp

```diff
@@ -39,7 +39,13 @@ using mlir::TypeStorageAllocator;
 ///
 /// This only prints a subset of all types in the dialect. Use `printNestedType`
 /// instead, which will call this function in turn, as appropriate.
-static LogicalResult customTypePrinter(Type type, AsmPrinter &os) {
+static LogicalResult customTypePrinter(Type type, AsmPrinter &os,
+                                       bool includeConst) {
+  if (includeConst && isConst(type)) {
+    os << "const.";
+    includeConst = false;
```

Comment:
Will go away with agg const change.

Reply:
To be clear, you're suggesting to always print `const` on every aggregate member that has also explicitly specified `const`? 

So you can have something like this:
```mlir
!firrtl.const.bundle<a: uint<2>, b: const.sint<4>>
```